### PR TITLE
adding a fix to parsing_user_query

### PR DESCRIPTION
**Pull request by maimorag**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-38196)

## Description
- adding a fix to parsing_user_queryL changing "from" to "attribute_timestamp"
since "from" didn't work as expected, and "timestamp" was deprecated https://www.circl.lu/doc/misp/automation/#post-attributesrestsearch

### CODE REVIEW

**Review by adi88d — APPROVED** [review]
looking good!
lets wait to the response from the customer and see if the issue solved

**Comment by adi88d** [comment]
need to fix the **test_parsing_user_query_remove_timestamp** unitest too

File: Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.py [hunk 1]
```diff
@@ -248,12 +248,11 @@ def parsing_user_query(query: str, limit: int, page: int = 1, from_timestamp: st
     try:
         params = json.loads(query)
         params["returnFormat"] = "json"
-        params.pop("timestamp", None)
```
Comment (dantavori) [inline Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.py:251]: why was `timestamp` originally popped out, and why do we remove this line now?

File: Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.py [hunk 2]
```diff
@@ -248,12 +248,11 @@ def parsing_user_query(query: str, limit: int, page: int = 1, from_timestamp: st
...
         if 'page' not in params:
             params["page"] = page
         params["limit"] = params.get("limit") or LIMIT
-        if from_timestamp:
-            params['from'] = from_timestamp
+        if not params.get("attribute_timestamp") or from_timestamp:
+            params['attribute_timestamp'] = from_timestamp
```
Comment (dantavori) [inline Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.py:255]: what if both `params.get("attribute_timestamp")` and `from_timestamp` are None?

File: Packs/FeedMISP/ReleaseNotes/1_0_33.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### MISP Feed
+
+- Fixed an issue where ***fetch-indicators*** where the query didn't pull updates.
```
Comment (dantavori) [inline Packs/FeedMISP/ReleaseNotes/1_0_33.md:6]:
```suggestion
- Fixed an issue in the ***fetch-indicators*** command where indicator updates were not pulled.
```

**Review by dantavori — COMMENTED** [review]
also test file needs to be fixed. is there a way to add a unittest for your change?

File: Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.py [hunk 4]
```diff
@@ -232,8 +232,10 @@ def build_params_dict(tags: List[str], attribute_type: List[str], limit: int, pa
         'limit': limit,
         'page': page
     }
+    if params.get("timestamp"):
+        params['attribute_timestamp'] = params.pop("timestamp")
```
Comment (dantavori) [inline Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.py:236]: params is defined above this condition and you can see it never has a `timestamp` item

**Review by dantavori — APPROVED** [review]
nice

**Pull request merged**